src/test/unit_tests/test_bucketspace_versioning.js

```diff
@@ -8,41 +8,34 @@ const path = require('path');
 const http = require('http');
 const mocha = require('mocha');
 const { S3 } = require('@aws-sdk/client-s3');
```

Comment:
If you can leave the `const { S3 } = require('@aws-sdk/client-s3');` and `const { NodeHttpHandler } = require('@smithy/node-http-handler');` near.